pkgs/development/tools/rust/cargo-tauri/beta.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
```

Comment:
Move this to `pkgs/ca/cargoi-tauri/package.nix`.

P.S.: have you tested this package? How?